src/index.html

```diff
@@ -15,8 +15,82 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Play&family=Roboto+Condensed:wght@500&family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="navigation">
+        <a href="#">
+          <img
+            class="img"
+            alt="logo"
+            src="./images/logo.png"
+          />
+        </a>
+        <nav class="nav-bar">
+          <a
+            href="#"
+            class="link is-active"
+          >
+            Apple
+          </a>
+          <a
+            href="#"
+            class="link"
+          >
+            Samsung
+          </a>
+          <a
+            href="#"
+            class="link"
+          >
+            Smartphones
+          </a>
+          <a
+            href="#"
+            class="link hover"
```

Comment:
The 'hover' class is applied to the 'Laptops & Computers' link. Ensure that this class is defined in your CSS and is intended for this specific link. If it is meant to be a hover effect for all links, consider applying it consistently across all navigation links.